client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button, Input, Typography } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Box } from '@mui/system';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+import useStyles from './useStyles';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { useState } from 'react';
+
+interface ProfilePhotoProps {
+  header: string;
+  currentUser?: User;
+  currentProfile?: any;
```

Comment:
Please avoid using `any` as a type. Let's use the Profile interface here instead (we can create one if it doesn't exist)